go/libkb/interfaces.go

```diff
@@ -564,6 +564,14 @@ type UserChangedHandler interface {
 	HandleUserChanged(uid keybase1.UID) error
 }
 
+type ConnectivityMonitorResult int
+
+const (
+	ConnectivityMonitor_YES ConnectivityMonitorResult = iota
+	ConnectivityMonitor_NO
+	ConnectivityMonitor_UNKNOWN
```

Comment:
Not sure why it didn't when I committed, I'll have to double check that is working for me